.env

```diff
@@ -148,6 +148,9 @@ HIP_DB_NAME=hipservice
 HIP_DB_HOST=db
 HIP_DB_PORT=5432
 OTP_WHITELIST_NUMBERS=""
+AUTHENTICATION_METHOD="Jwt"
```

Comment:
If we set Authentication scheme to `Jwt` will the cookie based auth still work for requests from Bahmni?